derived_images/salt/salt-minion/salt-minion.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Can you add some documentation explaining why we need this bash script (the whole thing about avoiding zombie processes)